clab/clab.go

```diff
@@ -642,12 +596,52 @@ func (c *CLab) GetNodeRuntime(contName string) (runtime.ContainerRuntime, error)
 // VethCleanup iterates over links found in clab topology to initiate removal of dangling veths
 // in host networking namespace or attached to linux bridge.
 // See https://github.com/srl-labs/containerlab/issues/842 for the reference.
-func (c *CLab) VethCleanup(_ context.Context) error {
+func (c *CLab) VethCleanup(ctx context.Context) error {
+	// remove all the links
 	for _, link := range c.Links {
-		err := c.RemoveHostOrBridgeVeth(link)
+		err := link.Remove(ctx)
 		if err != nil {
-			log.Infof("Error during veth cleanup: %v", err)
+			// if error log and continue
+			log.Error(err)
```

Comment:
TODO: link removal seems to be unimplemented

Reply:
I was unable to find a case in which the explicit removal of a link was required.
On removal of the netns, the interfaces are also cleaned up. I did not remove the Remove() function so far.
Can you point out a case in whcih the remove would be needed explicitely?